pkg/cli/mirror/copy.go

```diff
@@ -307,18 +367,18 @@ func (o *MirrorOptions) bulkImageMirror(isc *v1alpha2.ImageSetConfiguration, des
 		if operator.TargetTag != "" {
 			to += ":" + operator.TargetTag
 		}
-		err = pushImage(operator.Catalog, to, dstSkipTLS, insecureSigPolicy, remoteRegFuncs)
+		err = pushImage(operator.Catalog, to, o.DestSkipTLS, o.OCIInsecureSignaturePolicy, remoteRegFuncs)
 		if err != nil {
 			return err
 		}
 	}
-
-	err := remoteRegFuncs.mirrorMappings(*isc, mapping, dstSkipTLS)
+	err := remoteRegFuncs.mirrorMappings(*isc, mapping, o.DestSkipTLS)
```

Comment:
All of this: 
```golang
err := remoteRegFuncs.mirrorMappings(*isc, mapping, o.DestSkipTLS)
if err != nil {
    return err
}

return nil
```
you can easily replace that with simple:
```golang
return remoteRegFuncs.mirrorMappings(*isc, mapping, o.DestSkipTLS)
```
less code, more readable :smile: